cmd/podman/containers/stats.go

```diff
@@ -168,7 +170,7 @@ func outputStats(reports []define.ContainerStats) error {
 	if report.IsJSON(statsOptions.Format) {
 		return outputJSON(stats)
 	}
-	format := "{{.ID}}\t{{.Name}}\t{{.CPUPerc}}\t{{.MemUsage}}\t{{.MemPerc}}\t{{.NetIO}}\t{{.BlockIO}}\t{{.PIDS}}\n"
+	format := "{{.ID}}\t{{.Name}}\t{{.CPUPerc}}\t{{.MemUsage}}\t{{.MemPerc}}\t{{.NetIO}}\t{{.BlockIO}}\t{{.PIDS}}\t{{.UpTime}}\t{{.AVGCPU}}\n"
```

Comment:
I don't understand why commit 1d36ad56bd2714ee482539dc3983ad1232d293f1 is backported?  This is changing the default podman stats output which IMO should never be done for backports/minor releases.

Reading the bug the request is only to fix the broken CpuPerc calculation not adding new features. I would expect a backport of only afb72eaa849f577842e51de96a497406d3e87fea to be enough to fix the issue, of course you may need to rebase so it applies correctly to 3.0.

Reply:
The reason I included  1d36ad5 is that I initially grabbed both commits from#13597, and that included 130bcc3 which includes calculations on the stats.AvgCPU field  which was defined by 1d36ad5.  I wasn't thrilled about the change to the stats line either given the versions.  Let me go revisit the BZ and see if we can pull it off with just the one commit.  Thanks for the review.